frontend/ui/widget/textwidget.lua

```diff
@@ -41,9 +41,9 @@ function TextWidget:updateSize()
     if not tsize then
         self._length = 0
     else
-        self._length = tsize.x
+        self._length = math.ceil(tsize.x)
```

Comment:
It's the width of the text given by RenderText:sizeUtf8Text().
I admit there is probably some padding (at least 1 white pixel on each side) and that we could floor() it or round() it, but ceil()'ing it makes logically more sense to me: if it says text size = 123.5 pixels, better to report it needs 124 pixels rather than 123 for the layout to give it room, so that when it is painted, if the 124th pixel is black, it is not painted over the 1st pixel of the next widget area.